### PR TITLE
Add a preface to introduce terms; revise CF Council and introduce the EGC

### PR DESCRIPTION
**Pull request by ebullient**
@gavinking and @yrodiere .. does this suffice for an initial introduction of terms? 

There will need to be additional edits between Members & Council to round these out..

### CODE REVIEW

File: bylaws/0-preface.md
Comment (yrodiere) [inline bylaws/0-preface.md:40]: Might be on purpose, but just in case it's an oversight: missing link here.

File: bylaws/0-preface.md
Comment (yrodiere) [inline bylaws/0-preface.md:46]: Might be on purpose, but just in case it's an oversight: missing link here.

File: bylaws/2-cf-membership.md
Comment (yrodiere) [inline bylaws/2-cf-membership.md:20]: Maybe mention there's one and only one leader per project? Unless you don't want to enforce that :shrug:

File: bylaws/3-cf-council.md
Comment (yrodiere) [inline bylaws/3-cf-council.md:108]: I'd but a `Composition` section here, to be explicit about the fact that EGC = CFC + Project leaders, with equal votes. I know it's already in the preface, but this section should probably be self-sufficient?

File: bylaws/3-cf-council.md
Comment (yrodiere) [inline bylaws/3-cf-council.md:111]: I still think that if you use the term "supermajority" you need to specify what that supermajority is (two-third, three-fifth, ...), or who has the power to define what it is and when (on a per-vote basis, periodically by "decree", ...).

It might be useful to mention whether the (super/simple) majority is based on the number of voting members or on the number of members able to vote, though.
Also, I think I saw something about what happens when not enough members are present in the CFC: we'd need to specify whether the same applies to the EGC.

**Review by yrodiere — COMMENTED** [review]
Thanks, sounds good to me. I added a few comments, though I suppose the ones about voting would warrant a PR to better explain what I mean. I can do that after this gets merged, if you want.

File: bylaws/0-preface.md
Comment (gavinking) [reply bylaws/0-preface.md:46]: Naaaa, these are perfectly fine implicit markdown links.

**Comment by gavinking** [comment]
LGTM

File: bylaws/0-preface.md
Comment (yrodiere) [reply bylaws/0-preface.md:46]: Hmm, okay. I wasn't familiar with this syntax.

**Comment by ebullient** [comment]
> Thanks, sounds good to me. I added a few comments, though I suppose the ones about voting would warrant a PR to better explain what I mean. I can do that after this gets merged, if you want.

Comments about voting should go into decision making or to specific member/cf-council pages. So yes, let's make them, but we don't necessarily need it here.

File: bylaws/2-cf-membership.md
Comment (ebullient) [reply bylaws/2-cf-membership.md:20]: I think there should be a cap... but I don't know that the cap is one. Another conversation for another PR. I was at least trying to toss the project definition here to provide a scope for Project leader / Code Owner definition.

Comment (ebullient) [reply bylaws/2-cf-membership.md]: Pushed this one (will squash and commit when we're all done). LMK what you think.

File: bylaws/3-cf-council.md
Comment (ebullient) [reply bylaws/3-cf-council.md]: I did update to make a few clarifications... it ended up being a bit big, but I think it is an improvement generally. LMK.

File: bylaws/2-cf-membership.md
Comment (yrodiere) [inline bylaws/2-cf-membership.md:27]: Typo here

```suggestion
**Code Owners:** The `CODEOWNERS` file in a GitHub project organization's repository specifies individuals responsible for code in a repository.  Nominated Project Leaders should be listed in their project's CODEOWNERS file. The CF wants to ensure that individuals representing the project on the EGC have a demonstrable commitment to that project's ongoing health and success.
```

File: bylaws/3-cf-council.md
Comment (yrodiere) [inline bylaws/3-cf-council.md:78]: Asynchronous meetings are a fine idea, but I'm not sure how it translates in practice :grin: 

```suggestion
The CFC uses electronic and (where possible) asynchronous methods for meetings and decision-making, catering to our globally distributed membership. This facilitates inclusive participation and effective governance.
```

File: bylaws/3-cf-council.md
Comment (yrodiere) [inline bylaws/3-cf-council.md:114]: Sounds good.
We'll probably need a mention somewhere that the EGC or CF members may have access to the Elekto platform to spontaneously organize a vote -- for this purpose in particular, but maybe also for others.

**Review by yrodiere — APPROVED** [review]
LGTM, thanks.

I added a few comments, but nothing that should prevent the PR from being merged.

File: bylaws/3-cf-council.md
Comment (ebullient) [reply bylaws/3-cf-council.md:114]: Yes.. we can do that w/ refinements to decision-making page and enhancements to automation to make those easier to organize.

**Review by yrodiere — APPROVED** [review]

**Review by kenfinnigan — APPROVED** [review]

File: bylaws/0-preface.md
Comment (aalmiray) [inline bylaws/0-preface.md:35]: Do we need a policy regarding security, CVEs, treat disclosures, etc?

File: bylaws/2-cf-membership.md
Comment (aalmiray) [inline bylaws/2-cf-membership.md:22]: Only one, or at least one?

File: bylaws/2-cf-membership.md
Comment (aalmiray) [inline bylaws/2-cf-membership.md:27]: This implies projects should follow certain guidelines and meet a minimum set of requirements in their file structure and contents. Such requirements must be defined and explicitly listed somewhere.

File: bylaws/3-cf-council.md
Comment (aalmiray) [inline bylaws/3-cf-council.md:69]: How do we handle potential conflicts in EGC membership. i.e, a project would like a different member to step up to the EGC but the current member does not want to let go for example.

File: bylaws/0-preface.md
Comment (ebullient) [reply bylaws/0-preface.md:35]: I do have a security policy draft.. I didn't include it here, I guess. Will fix.

File: bylaws/2-cf-membership.md
Comment (ebullient) [reply bylaws/2-cf-membership.md:22]: That's the question. How do I avoid swamping the single maintainer of a project by another project that has 5 leads?

File: bylaws/2-cf-membership.md
Comment (ebullient) [reply bylaws/2-cf-membership.md:27]: Yes. We can summarize. see `templates` .. I have to put a CODEOWNERS in there, but yes. we can clarify (I think in templates) what breadcrumbs/artifacts we expect projects to have. (and try to keep them light)

File: bylaws/3-cf-council.md
Comment (ebullient) [reply bylaws/3-cf-council.md:69]: That is another good question, aside from conflict resolution help, I'd expect this to be resolved by the project itself.  One of the things the foundation will ask for (between GOVERNANCE.md and CODEOWNERS) is a clear definition of who the project leaders (not just the EGC rep) are. 

A project could decide to adopt CF decision making processes (and maybe the automation that comes with it), in which case, votes are held, and access/representation changes, whether that person is a fan of the plan or not.

File: bylaws/0-preface.md
Comment (ebullient) [reply bylaws/0-preface.md:36]: Moved it [here](https://github.com/commonhaus/foundation-draft/pull/52/files#diff-bd665ab94decb701b58531759c1876e69e7b3888d056122ef6c2f988a96d1f9c) as a start.

File: bylaws/2-cf-membership.md
Comment (gavinking) [reply bylaws/2-cf-membership.md:22]: I think you could just say that each project gets one vote if that's the problem.

File: bylaws/2-cf-membership.md
Comment (ebullient) [reply bylaws/2-cf-membership.md:27]: I updated the templates/README.md to describe these files (as they are currently drafted).. Does that help?

File: bylaws/2-cf-membership.md
Comment (ebullient) [reply bylaws/2-cf-membership.md:26]: That was the intent. I suppose I could revise to use the word "representative" rather than leader. I think I'll do that. less confusing.

Comment (ebullient) [reply bylaws/2-cf-membership.md]: Updated. WDYT?

File: bylaws/2-cf-membership.md
Comment (ebullient) [reply bylaws/2-cf-membership.md]: Revised the language of this section to drop reference to CODEOWNERS, as that belongs elsewhere. I think it reads better now.

I also updated the description of projects to be less GitHub-centric.

**Comment by ebullient** [comment]
Merging for first pass legal review.

**Pull request merged**
